test/validate.spec.js

```diff
@@ -411,6 +418,36 @@ describe('screener-runner/src/validate', function() {
       });
     });
 
+    describe('validate.sauce', function() {
```

Comment:
@xwa130saucer can we add the flip case? "Should allow launchSauceConnect = true when using all sauce browsers"